datastore/datastore.py

```diff
@@ -404,6 +404,99 @@ def update_entity_data(self, entity_name, entity_data, language_script, **kwargs
                                                        language_script=language_script,
                                                        **kwargs)
 
+    def get_dictionary_supported_languages(self, dictionary_name, **kwargs):
```

Comment:
Naming for the entire file
`dictionary`-> `entity`
`word` -> `value`
`word_list` -> `values`

Reply:
All other crud methods
```
create
populate
repopulate
delete
get_entity_dictionary
delete_entity
update_entity_data
```

 are now more or less unnecessary or do wrong things that are incompatible with language supporting data structure. Should we let them be ?